### PR TITLE
Add Drone CI configuration

### PR DESCRIPTION
**Pull request by Menda**
Done and tested with this ticket: https://github.com/Humanitec/time_tracking_service/pull/6

### CODE REVIEW

File: service_builder/setup_service.py [hunk 1]
```diff
@@ -123,6 +123,40 @@ def _configure_docker(name_project):
     content = get_template_content(os.path.join('readme', 'docker.md'))
     append_to_file(file_readme, content)
 
+    PrettyPrint.msg_blue('Docker support was successfully added')
```
Comment (bogdangi) [inline service_builder/setup_service.py]: This line is not tested

File: service_builder/setup_service.py [hunk 2]
```diff
@@ -123,6 +123,40 @@ def _configure_docker(name_project):
...
+
+
+def _configure_drone_ci():
+    file_requirements = os.path.join('requirements', 'ci.txt')
+    content = get_template_content(file_requirements)
+    open(file_requirements, 'a').close()
```
Comment (bogdangi) [inline service_builder/setup_service.py]: This line is not tested

File: service_builder/setup_service.py [hunk 3]
```diff
@@ -123,6 +123,40 @@ def _configure_docker(name_project):
...
+    append_to_file(file_requirements, content)
+
+    file_flake8 = '.flake8'
+    content = get_template_content(os.path.join('drone-ci', file_flake8))
+    open(file_flake8, 'a').close()
```
Comment (bogdangi) [inline service_builder/setup_service.py]: This line is not tested

File: service_builder/setup_service.py [hunk 4]
```diff
@@ -123,6 +123,40 @@ def _configure_docker(name_project):
...
+    append_to_file(file_flake8, content)
+
+    file_coverage = '.coveragerc'
+    content = get_template_content(os.path.join('drone-ci', file_coverage))
+    open(file_coverage, 'a').close()
```
Comment (bogdangi) [inline service_builder/setup_service.py]: This line is not tested

File: service_builder/setup_service.py [hunk 5]
```diff
@@ -123,6 +123,40 @@ def _configure_docker(name_project):
...
+    append_to_file(file_coverage, content)
+
+    file_drone = '.drone.yml'
+    content = get_template_content(os.path.join('drone-ci', file_drone))
+    open(file_drone, 'a').close()
```
Comment (bogdangi) [inline service_builder/setup_service.py]: This line is not tested

File: service_builder/setup_service.py [hunk 6]
```diff
@@ -123,6 +123,40 @@ def _configure_docker(name_project):
...
+    append_to_file(file_drone, content)
+
+    file_tcp = 'tcp-port-wait.sh'
+    content = get_template_content(os.path.join('drone-ci', file_tcp))
+    open(file_tcp, 'a').close()
```
Comment (bogdangi) [inline service_builder/setup_service.py]: This line is not tested

File: service_builder/setup_service.py [hunk 7]
```diff
@@ -123,6 +123,40 @@ def _configure_docker(name_project):
...
+    append_to_file(file_tcp, content)
+    os.chmod(file_tcp, 0o755)
```
Comment (bogdangi) [inline service_builder/setup_service.py]: This line is not tested

File: service_builder/setup_service.py [hunk 8]
```diff
@@ -123,6 +123,40 @@ def _configure_docker(name_project):
...
+
+    PrettyPrint.msg_blue('Drone CI support was successfully added. Make sure '
```
Comment (bogdangi) [inline service_builder/setup_service.py]: This line is not tested

File: service_builder/setup_service.py [hunk 9]
```diff
@@ -134,7 +168,14 @@ def setup():
...
         'Type in the name of your application (e.g.: appointment):')
     _create_app(name_project, name_app)
+
     is_answer_yes = yes_or_no('Add Docker support?')
     if is_answer_yes:
         _configure_docker(name_project)
+
+    is_answer_yes = yes_or_no(
+        'Add Drone CI test support?')
+    if is_answer_yes:
```
Comment (bogdangi) [inline service_builder/setup_service.py]: This line is not tested

File: service_builder/setup_service.py [hunk 10]
```diff
@@ -134,7 +168,14 @@ def setup():
     name_app = get_input(
         'Type in the name of your application (e.g.: appointment):')
     _create_app(name_project, name_app)
+
     is_answer_yes = yes_or_no('Add Docker support?')
     if is_answer_yes:
         _configure_docker(name_project)
+
+    is_answer_yes = yes_or_no(
```
Comment (bogdangi) [inline service_builder/setup_service.py]: This line is not tested

File: service_builder/templates/drone-ci/.drone.yml [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+    image: python:3.7
+    commands:
+      - pip install -r requirements/ci.txt
+      - cat .flake8
+      - flake8 .
+      - ./tcp-port-wait.sh postgres 5432
+      - coverage run --source='.' manage.py test -v 2
+      - coverage report -m
+    environment:
+      - DATABASE_ENGINE=postgresql
```
Comment (jefmoura) [inline service_builder/templates/drone-ci/.drone.yml]: - Will we always use postgresql?
- Track is also a Django application and we use MongoDB

**Comment by bogdangi** [comment]
how about other parts which are not tested?

**Pull request merged**